app/javascript/components/patient/history/HistoryList.js

```diff
@@ -1,16 +1,16 @@
 import React from 'react';
 import _ from 'lodash';
 import { PropTypes } from 'prop-types';
-import { Card, Row } from 'react-bootstrap';
+import { Button, Card, Row } from 'react-bootstrap';
```

Comment:
Really small unimportant thing, but I appreciate the alphabetizing here. It keeps our code base neat and tidy.